index.html

```diff
@@ -145,9 +145,17 @@ <h2>Extensions to the <dfn data-cite="HTML5#the-window-object">Window</dfn> inte
       </pre>
     </section>
     <section>
-      <h2><dfn data-lt="ContinuousEvent">ContinuousEvents</dfn></h2>
-      <p>A task is considered to be a <code>ContinuousEvent</code> if the task will dispatch an event that is a <a>UIEvent</a> or <a>TouchEvent</a> or a child of either and if for events of this type the user agent typically fires multiple events of this type in succession at a rate that is implementation and platform specific.</p>
-      <p class="note">Generally the events that are considered to be continous are <dfn data-cite="uievents#event-type-mousemove">mousemove</dfn>, <dfn data-cite="uievents#event-type-wheel">wheel</dfn> and <dfn data-cite="touch-events#the-touchmove-event">touchmove</dfn> all of which are specified in such a way where the browser is encouraged to fire the events at a rate specific for the browser and platform. Events that are introduced in the future and similar should also probably be considered to be continous.</p>
+      <h2>ContinuousEvents</h2>
```

Comment:
One thought: perhaps later we can format this document to have some max chars per line (maybe 100?). It's generally done, though personally I have no problem reviewing in the current format, so no strong preference.